server/server.js

```diff
@@ -468,6 +469,30 @@ app.get(`${apiBaseUrl}/employer-sites/:id`,
     return res.json(result);
   }));
 
+app.get(`${apiBaseUrl}/employer-sites/:id/participants`,
+  keycloak.allowRolesMiddleware('health_authority', 'ministry_of_health'),
+  keycloak.getUserInfoMiddleware(),
+  asyncMiddleware(async (req, res) => {
+    // await validate(ParticipantQuerySchema, req.query);
```

Comment:
Forgotten comment.

Reply:
Ah, I need to add validation! I totally forgot, I'll get back to this on monday

Reply:
The validation required is just making sure the user has access to the site, right? should be the same as the site details endpoint?